draft-ietf-spice-sd-cwt.md

```diff
@@ -319,14 +319,43 @@ For example, the `inspector_license_number` claim is a Salted Disclosed Claim, c
     /value/  "ABCD-123456"
 ]>>
 ~~~
+{: title="CBOR extended diagnostic notation representation of inspector_license_number disclosure"}
+
+
+This is represented in CBOR pretty printed formal as follows (end of line comments and spaces inserted for clarity):
+
+~~~ cbor-pretty
+58 21                                  # bytes(33)
```

Comment:
@OR13 @mprorock @henkbirkholz
I just want to make sure that we still want the extra bstr encoding of each disclosure: 
`<<[salt, key, value]>>` vs `[salt, key, value]`. I don't have a strong objection, but I don't see the value in the extra encoding/decoding step (or the extra 2 or 3 bytes). I don't think the bstr encoding prevents the problem of the inside value being non-deterministically encoded.

Reply:
Yeah, I could be convinced on this.

My reasoning is similar to the feeling of processing COSE headers.

I decode a cose-sign1, and the protected header is ready to use as bytes, for example as aead aad.

Then if / when I need parameters inside I decode the protected header.

In our case, we need to compute hashes on these disclosures right away, so if they decode as arrays, then will need to be re-encoded to produce the hashes.

We maybe don't need the outer encoding now that we don't had sd_hash in kbt... so the decoded header would look like:

```
/ sd_claims / 17 : /just the disclosures chosen by the Holder/
<<[
    <<[
        /salt/   h'8d5c15fa86265d8ff77a0e92720ca837',
        /claim/  501,  / inspector_license_number /
        /value/  "ABCD-123456"
    ]>>,
    <<[
        /salt/   h'86c84b9c3614ba27073c7e5a475a2a13',
        /value/  1549560720  / inspected 7-Feb-2019 /
    ]>>,
    <<[
        /salt/   h'30eef86edeaa197df7bd3d17dd89cd87',
        /claim/  "region",
        /value/  "ca" /California/
    ]>>
]>>
```

Instead of:

```
/ sd_claims / 17 : / just the disclosures chosen by the Holder /
[
    <<[
        /salt/   h'8d5c15fa86265d8ff77a0e92720ca837',
        /claim/  501,  / inspector_license_number /
        /value/  "ABCD-123456"
    ]>>,
    <<[
        /salt/   h'86c84b9c3614ba27073c7e5a475a2a13',
        /value/  1549560720  / inspected 7-Feb-2019 /
    ]>>,
    <<[
        /salt/   h'30eef86edeaa197df7bd3d17dd89cd87',
        /claim/  "region",
        /value/  "ca" /California/
    ]>>
]
```